hcl/hcl_stack_test.go

```diff
@@ -181,6 +181,30 @@ func TestHCLParserStack(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "stack attributes supports tm_ funcalls",
+			input: []cfgfile{
+				{
+					filename: "stack.tm",
+					body: `
+						terramate {
+							required_version = ""
+						}
+						stack {
+							name = tm_upper("stack-name")
+						}
+					`,
+				},
+			},
+			want: want{
+				config: hcl.Config{
+					Terramate: &hcl.Terramate{},
+					Stack: &hcl.Stack{
+						Name: "STACK-NAME",
+					},
+				},
+			},
+		},
```

Comment:
```suggestion
		{
			name: "stack attributes supports tm_ funcalls",
			input: []cfgfile{
				{
					filename: "stack.tm",
					body: `
						stack {
							name = tm_upper("stack-name")
						}
					`,
				},
			},
			want: want{
				config: hcl.Config{
					Stack: &hcl.Stack{
						Name: "STACK-NAME",
					},
				},
			},
		},
```

The `terramate` block is unrelated to the test itself.